packages/scaffold-ui/src/modal/w3m-button/index.ts

```diff
@@ -33,17 +33,11 @@ export class W3mButton extends LitElement {
 
   @state() private isLoading = ModalController.state.loading
 
-  @state() private balanceVal = AccountController.state.balance
```

Comment:
Needed to remove these, when switching between different accounts & networks and if the balance is not fetched, the button shows the disconnected state. This should be handled properly

Reply:
This is resolved but not outdated, is it ok?